erikiado/jp2_online
# Fix loading script to actually save options for answers.

#### What's this PR do?
Fixes issue #36 

#### Where should the reviewer start?
estudios_socioeconomicos/load.py

#### How should this be manually tested?
run the script from django's shell:
```
python manage.py shell
from estudios_socioeconomicos.load import load_data
load_data()
```
Then manually check that the options appear in the admin interface.
#### Any background context you want to provide?
n/a


This template was adapted ~stolen~ from [Quickleft/Sprint.ly](https://quickleft.com/blog/pull-request-templates-make-code-review-easier/)
